cmd/release/minor/main.go

```diff
@@ -0,0 +1,60 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+
+	"github.com/aws/eks-distro/cmd/release/minor/projects"
+	"github.com/aws/eks-distro/cmd/release/utils/changetype"
+	"github.com/aws/eks-distro/cmd/release/utils/values"
+)
+
+func main() {
+	latestSupportedReleaseBranch, err := values.GetLatestSupportedReleaseBranch()
+	if err != nil {
+		log.Fatalf("getting the latest supported release branch to create new minor release: %v", err)
+	}
+
+	// Adds new release branch to SUPPORTED_RELEASE_BRANCHES
+	addedReleaseBranch, err := values.AddNextReleaseBranch()
+	if err != nil {
+		log.Fatalf("getting the new release branch: %v", err)
+	}
+	log.Printf("Added %s to SUPPORTED_RELEASE_BRANCHES", addedReleaseBranch)
+
+	prevReleaseBranch, nextReleaseBranch := string(latestSupportedReleaseBranch), string(addedReleaseBranch)
+
+	// Adds project files new release branch
+	projectFilesAddedCount, err := projects.CreateFilesAndDirectories(prevReleaseBranch, nextReleaseBranch)
+	if err != nil {
+		log.Fatalf("at the end: %v", err)
+	}
+	log.Printf("Generated %d project files", projectFilesAddedCount)
+
+	// Adds release directory and sets RELEASE values to "0"
+	err = createReleaseDirectoryAndFiles(nextReleaseBranch)
+	if err != nil {
+		log.Fatalf("at the end: %v", err)
```

Comment:
same as above, not sure what this message means

Reply:
same as above. I changed this in the next revision